docs/articles/sdk/configure-qtcreator.rst

```diff
@@ -0,0 +1,52 @@
+Configuring QtCreator to use the SDK
+====================================
+
+After the installation of the SDK and sourcing environment, there are couple of
+steps to do for running QtCreator in target environment. This roadmap assumes
+that SDK and QtCreator should be installed and environments are sourced already.
```

Comment:
"that the SDK and QtCreator are installed and the environment is sourced already."